src/erasure-code/shec/CMakeLists.txt

```diff
@@ -33,17 +19,13 @@ add_dependencies(ec_shec_generic ${CMAKE_SOURCE_DIR}/src/ceph_ver.h)
 set_target_properties(ec_shec_generic PROPERTIES VERSION 1.0.0 SOVERSION 1)
 install(TARGETS ec_shec_generic DESTINATION lib/erasure-code)
 
-# ec_shec_sse3
-set(shec_neon_srcs
-  ${shec_srcs}
-  erasure-code/jerasure/gf-complete/src/neon/gf_w4_neon.c 
-  erasure-code/jerasure/gf-complete/src/neon/gf_w8_neon.c 
-  erasure-code/jerasure/gf-complete/src/neon/gf_w16_neon.c 
-  erasure-code/jerasure/gf-complete/src/neon/gf_w32_neon.c 
-  erasure-code/jerasure/gf-complete/src/neon/gf_w64_neon.c)
-
+# ec_shec_neon
 #TODO:build libec_shec_neon library on an ARM machine
 if(ARM_NEON OR ARM_NEON2)
+  set(shec_neon_srcs
+    ${shec_srcs}
```

Comment:
shec_srcs can be an OBJECT library to avoid compiling them separately for the ec_shec_generic and ec_shec_neon targets